src/main/java/org/alliancegenome/curation_api/model/entities/GeneDiseaseAnnotation.java

```diff
@@ -23,7 +23,9 @@ public class GeneDiseaseAnnotation extends DiseaseAnnotation {
 
     @IndexedEmbedded(includeDepth = 1)
     @IndexingDependency(reindexOnUpdate = ReindexOnUpdate.SHALLOW)
-    @ManyToOne
+    @ManyToOne(cascade = {})
+    @org.hibernate.annotations.OnDelete(action = org.hibernate.annotations.OnDeleteAction.CASCADE)
+    @JoinColumn(foreignKey = @ForeignKey(name="fk_genedasubject"))
```

Comment:
Do we need to explicitly define the ForeignKeys?

Reply:
> Do we need to explicitly define the ForeignKeys?

Excplicitly naming the ForeingKeys with a human readable name makes it much easier and less error prone to do manual or flyway migrations on it afterwards should that need to happen, like in the [flyway migration for this PR](https://github.com/alliance-genome/agr_curation/pull/436/files#diff-248cb3e18ab5ac25b9a33cd8fa7e167f7c3087bb8f6116e66c3abe0b13ce8635), where the only way to figure out the actual name of the ForeignKey to manipulate was looking at my (local) postgres database, and I can only hope that seemingly randomly generate name stays the same accross all environments (which currently seems to be the case).